DataManagementSystem/Utilities/CatalogUtilities.py

```diff
@@ -0,0 +1,138 @@
+########################################################################
+# $HeadURL$
+########################################################################
+
+""" DIRAC FileCatalog utilities
```

Comment:
Yes, this is correct. Since the check is done on the client side now, there is no need of this utilities on the server side.